components/StateMultipleChoiceQuizPage.tsx

```diff
@@ -1,10 +1,57 @@
-import React from "react";
-import { Text, View } from "react-native";
+import React, { useEffect, useState } from "react";
+import { Text, View, Button } from "react-native";
+import { Question } from "../types/QuizQuestions";
+import { quizQuestions } from "../types/QuizQuestions";
 
-export default function StateMutipleChoiceQuizPage() {
-  return (
+export default function StateMultipleChoiceQuizApp() {
```

Comment:
Why was this name changed? Shouldn't it be `StateMultipleChoiceQuizPage`? Didn't we already establish the fact that the component name should be same as the file name? Why was this intentionally done?